src/fragments/start/getting-started/flutter/generate-model.mdx

```diff
@@ -1,68 +1,96 @@
 
-## Generate the models locally
+Your project has been successfully initialized and connected to the cloud!
 
-1. Click **Next: Test locally in your app** on the Data modeling screen to proceed
-2. On the local setup instructions page, you will be prompted with on-screen instructions to pull the data model according to the selected platform.
-    1. **Use Amplify CLI to pull the data model**
-        - You should have already installed the Amplify CLI in a previous step
-        - Copy and run the command shown in your project root. This command will initialize your current project with Amplify as well as generate the data models you will be using locally
+Some next steps:
+"amplify status" will show you what you've added already and if it's locally configured or deployed
+"amplify add <category>" will allow you to add features like user login or a backend API
+"amplify push" will build all your local backend resources and provision it in the cloud
+"amplify console" to open the Amplify Console and view your project status
+"amplify publish" will build all your local backend and frontend resources (if you have hosting category added) and provision it in the cloud
 
-            ```bash
-            amplify pull --sandboxId <UUID>
-            ```
-        - Make a note of the sandbox id, just in case!
-    2. As the rest of the steps will be covered by the tutorial, you can skip the remaining steps and just click **Deploy to AWS** on the top right
-3. In order to deploy a backend, you will need an AWS account. We will go over this in a later step when you’re ready to sync your app to the cloud. For now, keep the link to this page open!
+Pro tip:
+Try "amplify add api" to create a backend API and then "amplify push" to deploy everything
+```
 
-At this point, Amplify should have generated several directories and configuration files in your project for you. Let’s take a look at the generated GraphQL schema.
+4. Now, we can run `amplify add api` to create the data model.
+ 
+    - For the first prompt, select **GraphQL**
+    - Highlight **Authorization modes** and press `<Enter>`. Change it to **Amazon Cognito User Pool**.
+    - Select **Default configuration**, **Username**, then **No, I am done.**
+    - Enter `N` when prompted to **Configure additional auth types**
+    - Select **Continue**
+    - For your schema template, choose **Blank Schema**
+    - When prompted to edit the schema, enter `Y`
 
-From your project root, navigate to the `amplify/backend/api/<api-name>/` directory and open `schema.graphql` using a text editor of your choice. You should see the following.
+    Your `schema.graphql` file will open. Replace its contents with the following:
 
 ```graphql
-type Todo @model @auth(rules: [{ allow: public }]) {
-    id: ID!
-    name: String!
-    description: String
-    isComplete: Boolean!
-}
+type BudgetEntry @model @auth(rules: [{ allow: owner }]) {
+  id: ID!
+  title: String!
+  description: String
+  attachmentKey: String
```

Comment:
It storage is not planned as a part of this tutorial we could remove this field for simplicity

Reply:
Yep, removed